application/templates/vpa.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and (.Values.vpa).enabled (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1") -}}
```

Comment:
we should specify `kind` in capabilities check because `autoscaling.k8s.io` might exist but vpa crd might not